code-challenges.rb

```diff
@@ -21,8 +35,16 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def remove_vowels (string_word)#parentheses for readiblity
+    p string_word.delete 'A' 'E' 'I' 'O' 'U' 'a' 'e' 'i' 'o' 'u'
+
+end
 
 
+remove_vowels(album1)#parentheses for readiblity
+remove_vowels(album2)#parentheses for readiblity
+remove_vowels(album3)#parentheses for readiblity
+
```

Comment:
Really great. I like the parentheses for readability. 
A cool thing about Ruby is you can put all your strings together into one. `'aeiouAEIOU'`